code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -3347,4 +3347,93 @@
 	name = "Pumpkin Bread slice"
 	desc = "A slice of pumpkin bread."
 	icon_state = "pumpkinbreadslice"
-	bitesize = 2
\ No newline at end of file
+	bitesize = 2
+
+////////////////SLIDERS////////////////
+
+/obj/item/weapon/reagent_containers/food/snacks/slider
+	name = "slider"
+	desc = "It's so tiny!"
+	icon_state = "slider"
+
+/obj/item/weapon/reagent_containers/food/snacks/slider/New()
+	..()
+	reagents.add_reagent("nutriment", 2.5)
+	bitesize = 1.5
+
+/obj/item/weapon/reagent_containers/food/snacks/slider/synth
+	name = "synth slider"
+	desc = "It's made to be tiny!"
+
+/obj/item/weapon/reagent_containers/food/snacks/slider/xeno
+	name = "xeno slider"
+	desc = "It's green!"
+	icon_state = "slider_xeno"
+
+/obj/item/weapon/reagent_containers/food/snacks/slider/xeno/New()
+	..()
+	reagents.add_reagent("nutriment", 1)
+	bitesize = 2
+
+/obj/item/weapon/reagent_containers/food/snacks/slider/chicken
+	name = "chicken slider"
+	desc = "Chicken sliders? That's new."
+	icon_state = "slider_chicken"
+
+/obj/item/weapon/reagent_containers/food/snacks/slider/chicken/New()
+	..()
+	reagents.add_reagent("nutriment", 1)
+	bitesize = 2
+/
+/obj/item/weapon/reagent_containers/food/snacks/slider/carp
+	name = "carp slider"
+	desc = "I wonder how it tastes!"
+	icon_state = "slider_carp"
+
+/obj/item/weapon/reagent_containers/food/snacks/slider/carp/New()
+	..()
+	reagents.add_reagent("nutriment", 1)
+	reagents.add_reagent("toxin", 1)
+	bitesize = 2.5
+
+/obj/item/weapon/reagent_containers/food/snacks/slider/carp/spider
+	name = "spidey slidey"
+	desc = "I think there's still a leg in there!"
+	icon_state = "slider_spider"
+
+/obj/item/weapon/reagent_containers/food/snacks/slider/clown
+	name = "honky slider"
+	desc = "HONK!"
+	icon_state = "slider_clown"
+
+/obj/item/weapon/reagent_containers/food/snacks/slider/clown/New()
+	..()
+	reagents.add_reagent("honkserum", 2.5)
+	bitesize = 2.5
+
+/obj/item/weapon/reagent_containers/food/snacks/slider/mime
+	name = "quiet Slider"
+	desc = "..."
+	icon_state = "slider_mime"
+
+/obj/item/weapon/reagent_containers/food/snacks/slider/slippery
+	name = "slippery slider"
+	desc = "It's so slippery!"
+	icon_state = "slider_slippery"
+
+/obj/item/weapon/reagent_containers/food/snacks/slider/slippery/Crossed(atom/movable/O) //exactly the same as soap
+	if (istype(O, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = O
+
+		if (H.shoes && H.shoes.flags & NOSLIP)
+			return
+
+		H.stop_pulling()
+		H << "<SPAN CLASS='notice'>You slipped on the [name]!</SPAN>"

```

Comment:
No he has it right because The is only put in front of the things name if its not marked as improper and its referenced by [ref] the [name] is the same as using \the [src] or the [boigah.name]